src/font/fontlibrary.c

```diff
@@ -834,31 +834,28 @@ void LCUI_InitFontLibrary( void )
 		{ FONTDIR"msyh.ttc", "Microsoft YaHei", NULL }
 	};
 #else
-#define FONTDIR "/usr/share/fonts/"
-#define MAX_FONTFILE_NUM 4
+#define MAX_FONTFILE_NUM 3
 	struct {
 		const char *path;
 		const char *family;
 		const char *style;
 	} fonts[MAX_FONTFILE_NUM] = {
 		{
-			FONTDIR"truetype/ubuntu-font-family/Ubuntu-R.ttf",
+			Fontconfig_GetPath( "Ubuntu" ),
 			"Ubuntu", NULL
 		}, {
-			FONTDIR"opentype/noto/NotoSansCJK-Regular.ttc",
-			"Noto Sans CJK SC", NULL
-		}, {
-			FONTDIR"opentype/noto/NotoSansCJK.ttc",
+			Fontconfig_GetPath( "Noto Sans CJK SC" ),
 			"Noto Sans CJK SC", NULL
 		}, {
-			FONTDIR"truetype/wqy/wqy-microhei.ttc",
+			Fontconfig_GetPath( "WenQuanYi Micro Hei" ),
 			"WenQuanYi Micro Hei", NULL
 		}
 	};
 #endif
 
 	fontlib.font_cache_num = 1;
 	fontlib.font_cache = NEW( LCUI_FontCache, 1 );
+	ZEROSET( fontlib.font_cache, LCUI_FontCache );

```

Comment:
Do not need this code, Because `fontlib.font_cache[0] = FontCache();` will fill data.